account_asset_management/models/account_asset_line.py

```diff
@@ -179,11 +179,15 @@ def write(self, vals):
     def _setup_move_data(self, depreciation_date):
         self.ensure_one()
         asset = self.asset_id
+        journal = asset.profile_id.journal_id
+        sequence = journal.sequence_id.with_context(
+            ir_sequence_date=depreciation_date
+        )
         move_data = {
-            'name': asset.name,
+            'name': sequence.next_by_id(),
```

Comment:
@mpanarin at this stage, the move is not confirmed so we need to put name = '/' or even leave empty (and use the default which happens to be '/'

Then the code in the method post() will then get the proper sequence.